Included the subscribed channel list also.

ushukkla opened the pull request:
I have included the channel link, to which any user is being subscribed. Through this, he/she may directly add the video in his/her channel to the playlist, without browsing.

ushukla116 committed:
Commented the changes

can we use these changes

ushukla116 committed:
Included the channel list also.

I have included the channel link, to which any user is being subscribed. Through this, he/she may directly add the video in his/her channel to the playlist, without browsing.

ushukla116 committed:
Update main.js

ushukla116 committed:
Include CSS File

Added CSS File inject.css for further designing in the app.

ushukkla commented:
Included the CSS File (inject.css) in inject.js

ushukla116 committed:
Update inject.css

ushukla116 committed:
Update inject.css

ushukla116 committed:
Included the inject.css file

Include the inject.css in inject.js for designing the HTML elements.

nishnik commented on src/inject/inject.js:
```diff
@@ -40,6 +44,10 @@ function insertButton() {
     }
+    else if (document.location.href == "https://www.youtube.com/feed/channels") {
+        to_match = to_match.concat("yt-uix-tile-link yt-ui-ellipsis yt-ui-ellipsis-2 yt-uix-sessionlink      spf-link ", '"]');
+        LOC_HREF = 5;
```
I don't think this works. Have you checked it?

IMO there should be substring check like on line no 35.
https://github.com/Manu-shukla/Play_Next/blob/c4baf927cc5a065c8be96866085ae160f3612b52/src/inject/inject.js#L35

ushukkla replied:
But, locally it is working on my system. I think it is due to cookie being saved, I'll check it once.

nishnik replied:
Could you post a screenshot if it works on your laptop, even if it works for your personal channel IMO it won't work for every channel. Kindly check

ushukkla replied:
Ok, let me check.

ushukla116 committed:
Created Content.js file for script injectioning.

Created a new file content.js, through it we could specify the user that they must visit Youtube's site to make their own PlayList and the URL they are visiting is wrong, so visit YouTube.

ushukla116 committed:
Made changes with content.js.

Added functionality of alert message showing to the user if he/she visits any other site and clicks on the Play Next Extension.

ushukla116 committed:
Added Omnibox Keyword.

The omnibox is the keyword which is shown inside Chrome's address bar. That means if you type something related to playlist, instantly chrome address bar will show you to add the package of Youtube Playlist we have made.

ushukla116 committed:
Added the omnibox keyword.

You should be able to type 'playlist' inside the address bar. And then you can press 'tab' button on the address bar, just then extension is going to be added to the chrome instantly.
